cmd/ocm-backplane/config/config.go

```diff
@@ -0,0 +1,29 @@
+package config
+
+import (
+	"github.com/spf13/cobra"
+)
+
+const (
+	ProxyURLConfigVar = "proxy-url"
+	URLConfigVar      = "url"
+)
+
+func NewConfigCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "config",
+		Short: "Get or set backplane-cli configuration",
+		Long: `Get or set backplane-cli configuration variables.
+The location of the configuration file is gleaned from ~/.config/backplane/config.json or the 'BACKPLANE_CONFIG' environment variable if set.
+
+The following variables are supported:
+url         Backplane API URL
```

Comment:
Variables are a direct mapping of the configuration variables.
In the config file, we call it `url` hence we ought to name the variable the same.
If we change it to `backplane-url` it will be a breaking change and users will have trouble after upgrading.

Reply:
yeah, What I exact mean is the name in the config file, so I mention it is out of scope here.

Reply:
Understood, we could've done it earlier, but we're too deep down the road now, would break the existing functionality :)